src/components/transferAmountInputSection/transferAmountInputSection.tsx

```diff
@@ -124,8 +133,22 @@ const TransferAmountInputSection = ({
           </TouchableOpacity>
         )}
       />
+      {transferType === TransferTypes.RECURRENT_TRANSFER && (
```

Comment:
can we make it a selection instead of input...? 1 - 12 months selection....?

Reply:
@noumantahir what should we do in selection? Should it be number of days or months?

Reply:
@gamingumar perhaps start with days... suggestions...? @feruzm

Reply:
@gamingumar @noumantahir Every day, every week, every month and custom options in hours.

Reply:
@feruzm @noumantahir 
We Cannot set a transfer recurrence that is less than 24 hours

Reply:
updated